src/components/UI/Card/CardHeader/CardHeaderElements/CardHeaderElements.jsx

```diff
@@ -38,13 +41,15 @@ const CardHeaderElements = (props) => {
       {!penClicked ? (
         <>
           {!props.isViewMode && <FaPen onClick={penClickHandler} />}
-          <input
-            type="checkbox"
-            onChange={(event) => {
-              props.onCbChange();
-            }}
-            className="cb"
-          ></input>
+          {!location.startsWith("/card") && (
```

Comment:
Relying on hard-coded path is no good. You can better pass some specific property to define whether is is opened as separate pae or not.